src/pages/PolicyPage/PolicyPage.tsx

```diff
@@ -0,0 +1,224 @@
+                site experience more efficient may not function properly.that
+                make your site experience more efficient and may not function
+                properly.
+              </p>
+            </section>
+            <section>
+              <h6>Third-party disclosure</h6>
+              <p>
+                We do not sell, trade, or otherwise transfer to outside parties
+                your Personally Identifiable Information
+              </p>
+            </section>
+            <section>
+              <h6>Third-party links</h6>
+              <p>
+                We do not include or offer third-party products or services on
+                our website.
+              </p>
+            </section>
+            <section>
+              <h6>Google</h6>
+              <p>
+                Google&apos;s advertising requirements can be summed up by
+                Google&apos;s Advertising Principles. They are put in place to
+                provide a positive experience for users.
+                <a href="https://support.google.com/adwordspolicy/answer/1316548?hl=en">
+                  https://support.google.com/adwordspolicy/answer/1316548?hl=en
+                </a>
+              </p>
+              <p>We use Google AdSense Advertising on our website.</p>
+              <p>
+                Google, as a third-party vendor, uses cookies to serve ads on
+                our site. Google’s use of the DART cookie enables it to serve
+                ads to our users based on previous visits to our site and other
+                sites on the Internet. Users may opt-out of the use of the DART
+                cookie by visiting the Google Ad and Content Network privacy
+                policy.
+              </p>
+              <ul>
+                <b>We have implemented the following:</b>
+                <li>Remarketing with Google AdSense</li>
+                <li>Demographics and Interests Reporting</li>
+              </ul>
+            </section>
+            <section>
+              <h6>California Online Privacy Protection Act</h6>
+              <p>
+                CalOPPA is the first state law in the nation to require
+                commercial websites and online services to post a privacy
+                policy. The law&apos;s reach stretches well beyond California to
+                require any person or company in the United States (and
+                conceivably the world) that operates websites collecting
+                Personally Identifiable Information from California consumers to
+                post a conspicuous privacy policy on its website stating exactly
+                the information being collected and those individuals or
+                companies with whom it is being shared.{" "}
+                <span>
+                  – See more at:{" "}
+                  <a href="http://consumercal.org/california-online-privacy-protection-act-caloppa/#sthash.0FdRbT51.dpuf">
```

Comment:
please add `rel = "noopener noreferrer" we need to add it because of security concerns
https://clever-solution.com/everything-you-need-to-know-about-rel-noopener-noreferrer-tags-purpose-benefits-and-seo-impact/

Reply:
@pashedin   Added to the code